src/hfold/Parser.hs

```diff
@@ -0,0 +1,55 @@
+module Hfold.Parser where
+
+import Data.Monoid ((<>))
+import Data.Text (Text)
+import qualified Data.Text as T
+import Text.ParserCombinators.Parsec
+
+import Hfold.Types
+
+parseFile :: Text -> [Content]
+parseFile t = case parse parseContent "content" (T.unpack t) of
+    Left err -> []
+    Right xs -> xs
+
+parseContent :: Parser [Content]
+parseContent = do
+    c <- many (quoted <|> codeBlock <|> normal)
```

Comment:
Here are the docs for `<*`: https://hackage.haskell.org/package/base-4.8.2.0/docs/Control-Applicative.html#v:-60--42-

I remember it as "the arrow points at the thing you want to keep".